full_snapshot.proto

```diff
@@ -479,6 +479,17 @@ message RelationStatistic {
 
   int32 frozenxid_age = 30; // The age of frozen xact ID for this relation (relative to latest stable xid)
   int32 minmxid_age = 31; // The age of minimum multixact ID for this relation (relative to latest stable mxid)
+
+  // For infrequently updated stats
+  int32 relpages = 40;                 // Size of the on-disk representation of this table in pages (of size BLCKSZ)
+  int64 reltuples = 41;                // Number of live rows in the table. -1 indicating that the row count is unknown
+  int32 relallvisible = 42;            // Number of pages that are marked all-visible in the table's visibility map
+  int32 relfrozenxid = 43;             // All transaction IDs before this one have been replaced with a permanent (“frozen”) transaction ID in this table
```

Comment:
I think it would be best if we make this 64-bit in the snapshot and already attach the epoch value in the collector (inferred from the current TXID)

Reply:
ahh, totally forgot that we talked about this 🤦 will tweak 👍